From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nitish Sakhawalkar <nitsakh@icloud.com>
Date: Tue, 26 Mar 2019 11:28:05 -0700
Subject: fixme: Comment trace event macro

This broke the build at some point. Does it still? We should probably remove
this patch and find out!

diff --git a/src/node_internals.h b/src/node_internals.h
index 084ecfa73657d1958d7552baa896e170934639c8..cb388f3a9f12949fd3ecb0406f7b550f4ca5ca7c 100644
--- a/src/node_internals.h
+++ b/src/node_internals.h
@@ -375,10 +375,11 @@ class TraceEventScope {
   TraceEventScope(const char* category,
                   const char* name,
                   void* id) : category_(category), name_(name), id_(id) {
-    TRACE_EVENT_NESTABLE_ASYNC_BEGIN0(category_, name_, id_);
+    // TRACE_EVENT_NESTABLE_ASYNC_BEGIN0(category_, name_, id_);
+    (void) category_; (void)name_; (void)id_;
   }
   ~TraceEventScope() {
-    TRACE_EVENT_NESTABLE_ASYNC_END0(category_, name_, id_);
+    // TRACE_EVENT_NESTABLE_ASYNC_END0(category_, name_, id_);
   }
 
  private:
